.github/workflows/publish_tagged.yaml

```diff
@@ -4,12 +4,12 @@ on:
   workflow_dispatch:
   push:
     tags:
-      - "v[0-9]+.[0-9]+.[0-9]+(\\+**)?"
+      - 'v*'
```

Comment:
shouldn't this be `v.*`?

Reply:
It seems to work as is for our `release` workflow:
https://github.com/drand/drand/blob/3231638ca2731afc57b4fa9db212b3b2792b3f2b/.github/workflows/release.yml#L3-L7